tahoe_sites/migrations/0004_auto_20220324_0333.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.2.12 on 2022-03-24 03:33
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tahoe_sites', '0003_auto_20220223_1333'),
+    ]
```

Comment:
please revert this migration

let's make the least amount of migrations as long as we haven't removed `edx-organizations` fork from production

Reply:
acknowledged 👍🏼